test/uk/gov/hmrc/brm/controllers/BirthEventsControllerValidationLengthSpec.scala

```diff
@@ -67,6 +67,7 @@ class BirthEventsControllerValidationLengthSpec extends UnitSpec with OneAppPerT
       header(ACCEPT, result).get shouldBe "application/vnd.hmrc.1.0+json"
     }
 
+    // TODO is this correct?
```

Comment:
@chris-ianson @manishsw we need to review this unit test as this isn't correct anymore?